bin/reth/src/db/vectors.rs

```diff
@@ -0,0 +1,85 @@
+use eyre::Result;
+use proptest::{
+    arbitrary::Arbitrary,
+    prelude::{any_with, ProptestConfig},
+    strategy::{Strategy, ValueTree},
+    test_runner::TestRunner,
+};
+use reth_db::{table::Table, tables};
+
+const VECTORS_FOLDER: &str = "test-vectors";
+
+pub(crate) fn generate_db_vectors() -> Result<()> {
```

Comment:
I would move this to a test-vector generating subcommand, the `db` command is mostly for viewing/manipulating the db